README.md

```diff
@@ -1,10 +1,16 @@
-# powsybl-diagram-viewer
+# powsybl-network-viewer
 
-Typescript library to integrate a powsybl svg diagram in a javascript project. The library is built with the Vite bundler.
+Typescript library to integrate network viewers in a javascript project.
+This library contains three viewers: 
+- single-line-diagram-viewer, to integrate the SVG representing a graph of voltage levels within a network, the SVG and the corresponding metadata JSON file being generated by [powsybl-diagram](https://github.com/powsybl/powsybl-diagram)
+- network-area-diagram-viewer, to integrate the SVG representing single-line diagrams, the SVG and the corresponding metadata JSON file being generated by  [powsybl-diagram](https://github.com/powsybl/powsybl-diagram)
+- network-map-viewer, to display the substations / voltage levels on a map
+
+The library is built with the Vite bundler.
 Node from v18+ is required to build with Vite.
 
 Installation using npm:  
-'npm install @powsybl/diagram-viewer'
+'npm install @powsybl/network-viewer'
```

Comment:
Is there  a procedure to rename an npm package ? We should find a way to make projects using @powsybl/diagram-viewer move to @powsybl/network-viewer I guess

Reply:
It seems that you can't do that. But you could provide users with a deprecated message to tell them to use the new name: https://stackoverflow.com/questions/28371669/renaming-a-published-npm-module

Reply:
`npm deprecate my-project-name@"<= latest-version" "This project has been renamed. Install using new-project-name instead."`